src/T2DMap.h

```diff
@@ -165,7 +172,7 @@ class T2DMap : public QWidget
     bool mSizeLabel;
     bool isCenterViewCall;
     QString mHelpMsg;
-
+    MapInfoOverride* mapInfoOverride;
```

Comment:
Use a `std::unique_ptr` instead of a raw one and `std::make_unique()` for creating it. It'll automatically handle object deletion for you when it goes out of scope - because as it stands this will leak memory when the profile is closed and Mudlet is kept running.